handlers/handler-flowcontrol-qps/src/main/java/org/apache/servicecomb/qps/QpsControllerManager.java

```diff
@@ -211,6 +212,8 @@ public QpsControllerManager setGlobalQpsStrategy(String globalLimitKey, String g
   private AbstractQpsStrategy chooseStrategy(String globalConfigKey, Long limit, Long bucket,
       String strategyKey) {
     AbstractQpsStrategy strategy;
+    AbstractQpsStrategy customStrategy = SPIServiceUtils
```

Comment:
Maybe this code can be better refactored.

1. all strategies are SPI 
2. any implementation have a name
3. choose the target strategy by configured name and default to 'FixedWindow'

Reply:
` new TokenBucketStrategy(globalConfigKey, limit, bucket);`
`new FixedWindowStrategy(globalConfigKey, limit);`
has different parameters.

Reply:
Hidden parameters in implementation and I think will work.

Reply:
done